ocs_ci/framework/conf/ocp_version/ocp-4.19-config.yaml

```diff
@@ -10,6 +10,11 @@ DEPLOYMENT:
   ignition_version: "3.2.0"
 ENV_DATA:
   vm_template: "rhcos-9.6.20250121-0-vmware.x86_64"
-  acm_hub_channel: release-2.13
-  acm_version: "2.13"
-  submariner_version: "0.20.0"
+  acm_hub_channel: "release-2.14"
+  acm_version: "2.14"
+  submariner_version: "0.21.0"
+  # Below values needs to be changed when acm and submariner are GAed
+  submariner_source: "downstream"
+  submariner_release_type: "unreleased"
+  acm_hub_unreleased: true
+  oadp_version: "1.5"
```

Comment:
@prsurve we might need to add:

  subctl_version: "subctl-rhel9:v0.20"
  
  As well